### PR TITLE
standardized script for blend360 mouseflow

### PR DESCRIPTION
**Pull request by hafizasyraf72**

### CODE REVIEW

File: Clients/Blend360/Analytics Script/Dashboard Mouseflow Kickfire/blend360_dashboard_mouseflow_kickfire.sql [hunk 1]
```diff
@@ -29,137 +28,99 @@ INSERT INTO `x-marketing.blend360.dashboard_mouseflow_kickfire`  (
...
+  STRING_AGG( _entryurl, ", \n"  ) OVER(PARTITION BY _visitorid, DATE(_timestamp) ORDER BY _timestamp) AS _webActivityURL,    
+  #_fullurl AS _fullurl,
+  -- ROW_NUMBER() OVER(PARTITION BY _userid, DATE(_timestamp) ORDER BY _timestamp) _page_order
+FROM mouseflow_recording
+),
+mouseflow_pageviews AS (
+SELECT
+  *
+FROM info_mouseflow_pageviews
+JOIN (SELECT DISTINCT CONCAT(_visitorid, _timestamp) AS _key FROM mouseflow_recording WHERE _order = 1)
```
Comment (izztsfn) [inline Clients/Blend360/Analytics Script/Dashboard Mouseflow Kickfire/blend360_dashboard_mouseflow_kickfire.sql:88]: Create a new CTE to remove subquery here

File: Clients/Blend360/Analytics Script/Dashboard Mouseflow Kickfire/blend360_dashboard_mouseflow_kickfire.sql [hunk 2]
```diff
@@ -29,137 +28,99 @@ INSERT INTO `x-marketing.blend360.dashboard_mouseflow_kickfire`  (
...
-    `webtrack_ipcompany.webtrack_ipcompany`
-  WHERE
-    _isisp = 0 ) kickfire
-WHERE
-  blend360_mflow._ipAddr = kickfire._ipaddr
-  AND (LENGTH(blend360_mflow._domain) = 0
-    OR blend360_mflow._domain IS NULL);
\ No newline at end of file
+  blend360_mflow._name      = kickfire._name,
+  blend360_mflow._location  = CASE
```
Comment (izztsfn) [inline Clients/Blend360/Analytics Script/Dashboard Mouseflow Kickfire/blend360_dashboard_mouseflow_kickfire.sql:111]: I think it's better to create a new line and indent the CASE statement like this one (refer the right hand side)
image

**Review by izztsfn — CHANGES_REQUESTED** [review]
Just some minor formatting changes. Overall looks good already

**Comment by hafizasyraf72** [comment]
thanks for the review. done amend from my side @izztsfn

**Review by izztsfn — APPROVED** [review]
Noice

**Pull request merged**